### DISCUSSION TITLE
Very Slow FPS rate in spherical mirror distortion mode

### DISCUSSION THREAD

**Discussion by evilonod** [Stellarium/stellarium#3960] [Q&A]
I'm trying to use Stellarium in a planetarium projected onto the dome by a spherical mirror.   When I switch on the Spherical Mirror Distortion mode in the tools my frame rate drops from 60 FPS down to something like  1.5 FPS.   It's not like I don't have enough GPU/CPU power....     

Is that mode really supposed to be that slow?

**Comment by gzotti** [comment]
It was fixed a few days ago. Please download the latest beta.

**Reply by evilonod** [reply]
How does one download the latest beta?    I can get the source code but am not setup to compile it on Windows.    I'll try the Linux version.

**Reply by alex-w** [reply]
Please check https://github.com/Stellarium/stellarium-data/releases/tag/weekly-snapshot
